files/Aboot/boot0.j2

```diff
@@ -608,15 +643,20 @@ run_kexec() {
 
 secureboot_install() {
     if [ -e "$image_path" ]; then
-        warn "Image folder $image_path already exist, wiping..."
-        rm -rf "$image_path"
+        warn "Image folder $image_path already exist (likely regular install)"
```

Comment:
>likely regular install [](start = 54, length = 22)

How do you assume previous installation type (regular/secureboot)?

Is it possible:
1. regular install, and regular install again
2. or secureboot install and secureboot install again?

Reply:
It's not possible to reinstall the same image, it gets skipped.
`is_swi_installed` based on the boot mode will check if the image is already installed for it.
This is why the assumption is safe to make.